x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/common/get_pivot_dropdown_options.ts

```diff
@@ -93,41 +95,44 @@ export function getPivotDropdownOptions(
 
   const combinedFields = [...indexPatternFields, ...runtimeFields].sort(sortByLabel);
   combinedFields.forEach((field) => {
+    const rawFieldName = field.name;
```

Comment:
question: kibana index patterns field definition can contain a `displayName` properly. Should we use this one instead?

Reply:
discussed with Pete and leave this PR as is, we have a related discuss issue here: https://github.com/elastic/kibana/issues/84045